docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,265 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: lieutenant-api-user
+rules:
+- apiGroups:
+  - syn.tools
+  resources:
+  - clusters
+  - tenants
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: lieutenant-api-user
+roleRef:
+  kind: Role
+  name: lieutenant-api-user
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: api-access-synkickstart
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: api-access-synkickstart
+EOF
+----
+
+=== Create Lieutenant Objects: Tenant and Cluster
+
+. Prepare access to API, replace `MYUSER` with your GitLab.com user id
++
+[source,shell]
+----
+export LIEUTENANT_TOKEN=$(kubectl -n lieutenant get secret $(kubectl -n lieutenant get sa api-access-synkickstart -o go-template='{{(index .secrets 0).name}}') -o go-template='{{.data.token | base64decode}}')
+export LIEUTENANT_AUTH="Authorization: Bearer ${LIEUTENANT_TOKEN}"
+export LIEUTENANT_URL="lieutenant.${K3S_INGRESS_IP}.nip.io"
+export GITLABCOM_USERID="MYUSER"
+----
+
+. Create a *Lieutenant Tenant* via the API
++
+[source,shell]
+----
+TENANT_ID=$(curl -s -H $LIEUTENANT_AUTH -H "Content-Type: application/json" -X POST --data "{\"displayName\":\"My first Tenant\",\"gitRepo\":{\"url\":\"ssh://git@gitlab.com/${GITLABCOM_USERID}/mytenant.git\"}}" http://${LIEUTENANT_URL}/tenants | jq -r ".id")
+echo $TENANT_ID
+----
++
+TIP: If everything went well, the Lieutenant Operator created a new git repository under https://gitlab.com/${GITLABCOM_USERID}/mytenant, which will be used to store the configuration used by Commodore to create a catalog for a cluster.
+
+. Retrieve the registered Tenants via API and directly on the cluster
++
+[source,shell]
+----
+curl -H $LIEUTENANT_AUTH http://${LIEUTENANT_URL}/tenants
+kubectl -n lieutenant get tenant
+kubectl -n lieutenant get gitrepo
+----
+
+. Register a *Lieutenant Cluster* via the API
++
+[source,shell]
+----
+CLUSTER_ID=$(curl -s -H $LIEUTENANT_AUTH -H "Content-Type: application/json" -X POST --data "{ \"tenant\": \"${TENANT_ID}\", \"displayName\": \"My first Project Syn cluster\", \"facts\": { \"cloud\": \"local\", \"distribution\": \"k3s\", \"region\": \"local\" }, \"gitRepo\": { \"url\": \"ssh://git@gitlab.com/${GITLABCOM_USERID}/cluster-gitops1.git\" } }" http://${LIEUTENANT_URL}/clusters | jq -r ".id")
+echo $CLUSTER_ID
+----
++
+TIP: If everything went well, the Lieutenant Operator created a new git repository under https://gitlab.com/${GITLABCOM_USERID}/cluster-gitops1 which will be used to store the generated catalog of deployment files.
+
+. Retrieve the registered Clusters via API and directly on the cluster
++
+[source,shell]
+----
+curl -H $LIEUTENANT_AUTH http://${LIEUTENANT_URL}/clusters
+kubectl -n lieutenant get cluster
+kubectl -n lieutenant get gitrepo
+----
+
+== Kickstart Commodore
```

Comment:
"Commodore is the configuration generation component. It will be configured to generate configuration for your Lieutenant Cluster $CLUSTER_ID generated above"

Reply:
added